MultiPartyExample/MultiPartyViewController.swift

```diff
@@ -366,6 +414,8 @@ extension MultiPartyViewController : TVIRoomDelegate {
         logMessage(messageText: "Connected to room \(room.name) as \(room.localParticipant?.identity ?? "").")
         NSLog("Room: \(room.name) SID: \(room.sid)")
 
+        room.localParticipant?.delegate = self
```

Comment:
Possibly worth adding a comment about setting localParticipant.delegate within didConnect to not miss events.